src/manifolds/Rotations.jl

```diff
@@ -99,6 +99,24 @@ function check_vector(M::Rotations{N}, p, X; kwargs...) where {N}
     return check_point(SkewSymmetricMatrices(N), X; kwargs...)
 end
 
+@doc raw"""
+    embed(M::Rotations{N}, p, X)
+
+Embed the tangent vector `X` at point `p` in `M` from
+its Lie algebra representation (set of skew matrices) into the
+Riemannian submanifold representation
+
+The formula reads
+```math
+X_{\text{embedded}} = p * X
+```
+"""
+embed(::Rotations{N}, p, X) where {N}
```

Comment:
```suggestion
embed(::Rotations, p, X)
```

since you never need the `N` you can also just omit it here.